src/libsync/logger.cpp

```diff
@@ -19,6 +19,7 @@
 #include <QDir>
 #include <QStringList>
 #include <QThread>
+#include <QtGlobal>
```

Comment:
do we really need that?

Reply:
it's likely to be included by something else, but technically it's where `Q_OS_WIN` is defined and I don't like it to depend on implicit includes...